src/components/shared/editControls/EditControls.jsx

```diff
@@ -14,9 +14,15 @@ export function EditControls(props) {
     const [isRemovalModalOpen, setRemovalModalOpen] = useState(false);
     const [items, setItems] = useState([]);
 
+    const inputEl = useRef(null);
+
     function onRemovalModalClick() {
         setRemovalModalOpen(!isRemovalModalOpen);
     }
+    function removeItem() {
+        inputEl.current.closest('.justify-content-center').remove();
```

Comment:
please use props to remove items from recipies/menus whatever. I just dont wont to remove card from the ui but from the real data